h5grove/tornadoutils.py

```diff
@@ -94,8 +95,9 @@ class MetadataHandler(BaseHandler):
     """/meta/ endpoint handler"""
 
     def get_content(self, h5file: h5py.File, path: Optional[str]):
-        resolve_links = parse_bool_arg(
-            self.get_query_argument("resolve_links", None), fallback=True
+        resolve_links = parse_link_resolution_arg(
+            self.get_query_argument("resolve_links", None),
+            fallback=LinkResolution.ONLY_VALID,
```

Comment:
I chose to put the default to a more sensible behaviour (resolving only if valid and returning the link instead) rather than keeping the previous behaviour (equivalent to `LinkResolution.ALL`).